### PR TITLE
feat: Adds `rag_embeddings_and_index_with_alloydb.ipynb` using Gemini

### PR DESCRIPTION
**Pull request by tanyarw**
# Description

This notebook shows how to generate embeddings for text stored in AlloyDB, and retrieve information using similarity search. The retrieved results are used to augment responses genertaed by `gemini-1.0-pro`.

- [x] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [x] You are listed as the author in your notebook or README file.
  - [x] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [x] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [x] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: gemini/use-cases/retrieval-augmented-generation/rag_embeddings_and_index_with_alloydb.ipynb [hunk 1]
```diff
@@ -0,0 +1,1101 @@
+{
```
Comment (holtskinner) [inline gemini/use-cases/retrieval-augmented-generation/rag_embeddings_and_index_with_alloydb.ipynb:1]: <p>Can you make these library names be in code font?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (holtskinner) [inline gemini/use-cases/retrieval-augmented-generation/rag_embeddings_and_index_with_alloydb.ipynb:1]: <p>Make "Google Patents" a hyperlink and change all of the column names to be code font.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (holtskinner) [inline gemini/use-cases/retrieval-augmented-generation/rag_embeddings_and_index_with_alloydb.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #16.            connector.close()
</pre><p><code>connector</code> isn't used elsewhere in this function. Is this intentional?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (holtskinner) [inline gemini/use-cases/retrieval-augmented-generation/rag_embeddings_and_index_with_alloydb.ipynb:1]: <p>Put <code>CREATE EXTENSION</code> in code font</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (tanyarw) [reply gemini/use-cases/retrieval-augmented-generation/rag_embeddings_and_index_with_alloydb.ipynb:1]: <p>Passed it in the function args, thanks for spotting</p>

**Review by holtskinner — APPROVED** [review]

**Pull request merged**